internal/webhooks/machinehealthcheck.go

```diff
@@ -197,12 +197,5 @@ func (webhook *MachineHealthCheck) validateCommonFields(m *clusterv1.MachineHeal
 		)
```

Comment:
Let's please drop unhealthyConditions from test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

and replace it through a comment that we are intentionally not setting it to test that the field is optional.

This gives us some free happy path e2e test coverage

Reply:
Sure